Remove fairseq from export_llama

lucylq opened the pull request:
Summary: This is not being used anymore; remove it and simplify export_llama

Differential Revision: D87831086

larryliu0820 reviewed (approved):
Thank you for cleaning this up!

lucylq committed:
Remove fairseq from export_llama (#16052)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/16052

This is not being used anymore; remove it and simplify export_llama. Also remove language_llama, the only user of fairseq.

Reviewed By: larryliu0820

Differential Revision: D87831086
